src/itinerary/Itinerary.style.tsx

```diff
@@ -20,6 +20,11 @@ export const StyledItinerary = styled.div`
     padding-top: ${space.m};
   }
 
+  & .kirk-itinerary--headline {
```

Comment:
nit: to me, the "--" syntax is for modifiers only but here it targets the headline so I'd write `kirk-itinerary-headline`
I'm not sure about the `--highlightRoad` that follows so no big deal.

Reply:
Let's change this when we'll review this component following @wakooka design doc